### ISSUE TITLE
Multiple losses, average instead of sum in reporting?

### ISSUE THREAD

**Issue by carlthome** [fchollet/keras#2709]
I love that multiple outputs are reported individually now but could the combined loss be an average, rather than a sum? Basically just add `total_loss /= len(self.outputs)` just after [this line](https://github.com/fchollet/keras/blob/master/keras/engine/training.py#L595).

Or maybe make 'sum' or 'average' an optional parameter in model.compile() or something?

**Comment by xingdi-eric-yuan** [comment]
why not modify your loss weights

**Comment by carlthome** [comment]
Oh, right. Already exists! Thanks @xingdi-eric-yuan!
